### PR TITLE
Add hexagondata source and organization

### PR DESCRIPTION
**Pull request by eduardosi25**

### CODE REVIEW

**Comment by eduardosi25** [comment]
@googlebot I signed it!

File: sources/hexagondata_bluekai.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "id": "HEXAGONDATA_BK",
```
Comment (pfrisella) [inline sources/hexagondata_bluekai.json:2]: ID should match the filename

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: sources/hexagondata_bluekai.json (hunk 1 above)
Comment (eduardosi25) [reply sources/hexagondata_bluekai.json]: ok, done! thanks we are still pending

**Comment by pfrisella** [comment]
Can you not use the existing Salesforce data source? and for Bluekai this seems like it should just be BlueKai by Oracle?

**Comment by eduardosi25** [comment]
> Can you not use the existing Salesforce data source? and for Bluekai this seems like it should just be BlueKai by Oracle?

Hi pfrisella:

In both cases, one thing we bring to the table is the ability to compare audiences across time. We save a weekly snapshot to see how the composition of each audience changes in time. This is a type of insight not available in the DMPs.

Salesforce: Salesforce does not have segments API. This is the only way to automate the export of audience profile information for Salesforce Audience Studio. The Salesforce Data source exports raw data which, because of the volume, does not make it feasible to create reports from that data. It is a very large volume of data that would require restructuring in a BI.

Oracle Bluekai: We do 3rd party data cleansing to make sure the reports only use high quality data sources.

**Comment by pfrisella** [comment]
I understand but this is simply a data registry for underlying data sources, independent of connectors and not for connector specific versions or transformations of other data sources. For example, if there are multiple connectors that query Twitter data, they all use the same TWITTER source, there isn't a data registry entry for each connector version of Twitter data. Also, consider users that are not familiar with your company and want to search for connectors that can query their salesforce data, it's likely they would search for "salesforce". They wouldn't be aware of your specific salesforce version of data so it is in your interest to use the existing salesforce entry.

**Comment by eduardosi25** [comment]
> I understand but this is simply a data registry for underlying data sources, independent of connectors and not for connector specific versions or transformations of other data sources. For example, if there are multiple connectors that query Twitter data, they all use the same TWITTER source, there isn't a data registry entry for each connector version of Twitter data. Also, consider users that are not familiar with your company and want to search for connectors that can query their salesforce data, it's likely they would search for "salesforce". They wouldn't be aware of your specific salesforce version of data so it is in your interest to use the existing salesforce entry.

ok, If I understood correctly, then the name of the connector should be Salesforce and not Hexagon data?

**Comment by eduardosi25** [comment]
> This doesn't represent your connector and is not the name of your connector. It is just a registry of data sources, not connectors. What I'm suggesting is that you don't need to add anything for salesforce, there already is an existing data source entry that you can use. The only thing you need to add is a source for BlueKai by Oracle since that doesn't exist. There is already an Oracle org entry so just add a new BLUEKAI source and that's it.
> 
> I suspect you're trying to submit a connector for publishing. In the connector manifest there is a `sources` property and you just need to include which underlying source(s) your connector queries data from. You can name your connector whatever you want in the manifest using the `name`, this is not dependent on which sources you list. See list item #4 on https://developers.google.com/datastudio/connector/pscc-requirements#datastudio. Then for your connector you would have `sources: ['SALESFORCE', 'BLUEKAI']` which then makes it easy to index what type of data your connector has available.

ok, I understand, we precisely want to register our source, the connection of the connectors is not directly to salesforce or bluekai, it is directly to our apis, our internal users would generate reports from our apis and would look for our connectors in the list, I regret not having explained us correctly, but thanks for clarifying these points.
in the manifest document our data source would be: "sources": ["HEXAGONDATA_BK"], for the example of our data, is the implementation correct?

**Comment by pfrisella** [comment]
There are many connectors which do not query directly to the source, use their own APIs (e.g. not directly to salesforce), but they still use the generic version of the data source. The point is not to index a specific version of salesforce or bluekai data. This just represents the underlying data source, whether it is modified or improved does not need to be specified here. You can explain that in your connector and associated learn more pages for your connector. This source just helps a user understand the type of data sources available to query.

So my recommendation is just to add a source for BlueKai by Oracle and use the existing Salesforce source. This will not have any negative affect on your connector or make it less searchable (it will actually make it easier for potential customers to find your connector) and it will allow you to move on with the process.

**Comment by pfrisella** [comment]
Closing due to inactivity. If this is still applicable send a new PR, but consider using existing data sources where applicable, as discussed above.

**Pull request closed without merging**